Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "rails", "6.1.4.4"
```

Comment:
Should we also be adding `sprockets-rails` as an explicit dependency here?

From the [upgrade guide](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#sprockets-is-now-an-optional-dependency):
> **Sprockets is now an optional dependency**
> The gem rails doesn't depend on sprockets-rails anymore. If your application still needs to use Sprockets, make sure to add sprockets-rails to your Gemfile.

It looks like `sprockets` is still pulled in as a dependency of the `sassc-rails` gem, so maybe that's good enough..? 🤔

Reply:
That's a good shout. As it's an explicit dependency of the app, we should call it out - especially as we use it for more than sassc.

What's also interesting is that this app uses Sprockets 4, whereas most of our apps don't because govuk_publishing_components requires sprockets < 4 and nearly all GOV.UK apps use that: https://github.com/alphagov/govuk_publishing_components/pull/1172.